docs/configuration.md

```diff
@@ -285,13 +285,13 @@ Apart from these, the following properties are also available, and may be useful
   </td>
 </tr>
 <tr>
-  <td><code>spark.files.userClassPathFirst</code></td>
+  <td><code>spark.driver.userClassPathFirst, spark.executor.userClassPathFirst</code></td>
```

Comment:
I agree with @sryza. If we're worried about duplicating description we can always say `Same as spark.driver.userClassPathFirst, but for the executors` or something.